### PR TITLE
Aarondonahue/lc testing

### PR DESCRIPTION
**Pull request by AaronDonahue**
Update machine config files for Livermore Computing to facilitate running E3SM tests from CIME.

Also fixed an issue in the Yaml CMAKE file that broke the FSCREAM-SA tests when run in CIME.

### CODE REVIEW

**Comment by AaronDonahue** [comment]
@bartgol , I tagged you as the reviewer for the one small change to the YAML cmake file in SCREAM.  Otherwise, the other changes are just machine config changes for Livermore Computing and shouldn't need changing.

File: components/scream/cmake/YamlCpp.cmake [hunk 1]
```diff
@@ -11,7 +11,7 @@ get_property(IS_YAMLCPP_ALREADY_BUILT GLOBAL PROPERTY E3SM_YAMLCPP_ALREADY_BUILT
 if (NOT IS_YAMLCPP_ALREADY_BUILT)
   set (YAMLCPP_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../externals/yaml-cpp)
   set (YAMLCPP_BIN_DIR ${CMAKE_BINARY_DIR}/externals/yaml-cpp)
-  message("YAML SRC DIR: ${YAMLCPP_SRC_DIR}")
```
Comment (bartgol) [inline components/scream/cmake/YamlCpp.cmake]: Thanks. Leftover from when things were not working, and I needed to debug.

**Comment by bartgol** [comment]
@AaronDonahue You should add more reviewers. The AT will automerge this PR, but I only reviewed the small yaml change. I have no idea if the rest makes sense, it's not my expertise. You should add someone that can verify it's sound.

**Comment by AaronDonahue** [comment]
@bartgol, thanks for suggesting that the rest be reviewed.  In this case they are just machine file changes for LC and I am the POC for LC so theoretically as long as I give the ok they should be alright.

**Pull request merged**